pysignalscope/scope.py

```diff
@@ -137,8 +164,18 @@ def modify(self, channel_data_factor: Optional[float] = None, channel_data_offse
                     index_list_to_remove.append(count)
             self.channel_time = np.delete(self.channel_time, index_list_to_remove)
             self.channel_data = np.delete(self.channel_data, index_list_to_remove)
+            modify_flag = True
+
         if channel_linestyle is not None:
             self.channel_linestyle = channel_linestyle
+            modify_flag = True
+        pass
+
+        # Log flow control ASA
+        logging.debug("%s :FlCtl", self.modulename)
```

Comment:
i prefer `f""` instead of ``%s`` to unify the code, as it is better readable.
There are more ``%s`` in the rest of the code...

Reply:
I use C-code formating style and prefer this. However your argument is not bad, especially the interpreter decide, which type of variable is to display. This decrease the probability of bugs, but you loose a little bit of control.
Conclusion: I agree and update it according your request.